src/main.js

```diff
@@ -7,6 +7,8 @@ import App from './App'
 import router from './router'
 import { database } from '@/services/firebase'
 import VueAnalytics from 'vue-analytics'
+import firebase from './services/firebase'
+
```

Comment:
You don't need this import.

This line  `import { database } from '@/services/firebase'` already import the database service from firebase services. The '@' character is a node alias to src =)

Reply:
Done